lib/lexer_top.mll

```diff
@@ -12,7 +12,7 @@ rule token = parse
  | _ as c        { failwith (Printf.sprintf "unexpected character '%c'. Did you forget a space after the '#' at the start of the line?" c) }
 
 and phrase acc buf = parse
-  | ("\n"* as nl) "\n  "
+  | ("\n"* as nl) "\n" ("  " | ws ws?)
```

Comment:
Shouldn't this be `("  " | "\t")` ?

Reply:
I wrote it like this because before 4.08 we need the parser to consume exactly 2 spaces to preserve the indentation of the rest of the line, and the `ws ws?` case is when there is a mix of tabs and spaces so we are not able to preserve the indentation in the same way, so we can consume all spaces in this case.

Reply:
`ws ws?` can also match a single space character.
There is a rendering problem in my recommendation above, there should be 2 spaces in the first case:
```
("  " | "\t")
```

Reply:
Looks like a single tabulation is enough indeed, done.